azure-pipelines.yml

```diff
@@ -0,0 +1,71 @@
+# Node.js with Angular
+# Build a Node.js project that uses Angular.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+variables:
+  appDir: 'fakebook/'
+
+trigger:
+  branches:
+   include: 
+     - main
+  paths:
+    include:
+      - az-pipeline-angular.yml
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+stages:
+  - stage: build
+    jobs:
+    - job: build
+      variables:
+        imageName: 'fakebookng'
+      
+      steps:
+      - task: Docker@2
+        displayName: Build Docker Image
+        inputs:
+          repository: $(imageName)
+          command: 'build'
+          Dockerfile: '$(appdir)Dockerfile'
+  
+  - stage: SonarCloud
```

Comment:
this stage can be `dependsOn: []` so it runs in parallel with the docker

Reply:
added